localstack/services/sqs/provider.py

```diff
@@ -288,6 +326,17 @@ def default_attributes(self) -> QueueAttributeMap:
             QueueAttributeName.SqsManagedSseEnabled: "false",
         }
 
+    def update_delay_seconds(self, value: int):
+        """
+        For standard queues, the per-queue delay setting is not retroactive—changing the setting doesn't affect the delay of messages already in the queue.
+        For FIFO queues, the per-queue delay setting is retroactive—changing the setting affects the delay of messages already in the queue.
+
+        https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-delay-queues.html
+
+        :param value: the number of seconds
+        """
+        self.attributes[QueueAttributeName.DelaySeconds] = str(value)
```

Comment:
Just checking, is there a reason why we accept an int, cast it to string, and cast it back to int at line 381? Never questioned that until now, but we seem to do this on multiple occasions, does this construct require strings?

Reply:
yea we keep the attributes in string types because this is what AWS returns when querying attributes. we could invert the logic of course, to convert to strings when we return the attributes. guess we can do that together with a general cleanup of the provider :+1: